### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by anonrig** [iyobo/koa-ts-controllers#16]
Hi!

First of all congrats on this beautiful project. I didn't want to bother with a bug I've been facing, but here's the truth:

```
path /Users/yagiz/Workspace/alfred/src/**/*/controller.ts
(node:44681) UnhandledPromiseRejectionWarning: /Users/yagiz/Workspace/alfred/src/admin/account/controller.ts:1
```

I'm receiving this error on the following code:
```javascript
  await bootstrapControllers(app, {
    router,
    basePath: '/',
    controllers: [__dirname + '/**/*/controller.ts'],
    initBodyParser: true,
    boomifyErrors: true,
    disableVersioning: true
  })
```

My .babelrc is:
```
{
  "presets": [["env", { "modules": false }], "@babel/preset-typescript"],
  "plugins": [
    "@babel/plugin-transform-modules-commonjs",
    ["@babel/plugin-proposal-decorators", { "legacy": true }],
    "@babel/proposal-class-properties",
    "@babel/proposal-object-rest-spread"
  ]
}
```

And my tsconfig.json is:
```json
{
  "compileOnSave": false,
  "exclude": [
    "node_modules",
    "docs",
    "migrations",
    "seeders",
    "tests",
    "websites",
    "dist"
  ],
  "include": ["./**/*.ts"],
  "compilerOptions": {
    /* Basic Options */
    "incremental": true /* Enable incremental compilation */,
    "target": "ES5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
    "lib": [
      "ES5",
      "ES2016",
      "ES2017"
    ] /* Specify library files to be included in the compilation. */,
    "allowJs": true /* Allow javascript files to be compiled. */,
    "checkJs": true /* Report errors in .js files. */,
    // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
    "declaration": true /* Generates corresponding '.d.ts' file. */,
    // "declarationMap": true /* Generates a sourcemap for each corresponding '.d.ts' file. */,
    "sourceMap": true /* Generates corresponding '.map' file. */,
    // "outFile": "./",                       /* Concatenate and emit output to single file. */
    // "outDir": "./dist" /* Redirect output structure to the directory. */,
    "rootDir": "./" /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */,
    // "composite": true /* Enable project compilation */,
    // "tsBuildInfoFile": "./",               /* Specify file to store incremental compilation information */
    "removeComments": true /* Do not emit comments to output. */,
    // "noEmit": true /* Do not emit outputs. */,
    "importHelpers": true /* Import emit helpers from 'tslib'. */,
    "downlevelIteration": true /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */,
    // "isolatedModules": true /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */,

    /* Strict Type-Checking Options */
    // "strict": true /* Enable all strict type-checking options. */,
    // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
    // "strictNullChecks": true /* Enable strict null checks. */,
    // "strictFunctionTypes": true /* Enable strict checking of function types. */,
    "strictBindCallApply": true /* Enable strict 'bind', 'call', and 'apply' methods on functions. */,
    // "strictPropertyInitialization": true,  /* Enable strict checking of property initialization in classes. */
    // "noImplicitThis": true /* Raise error on 'this' expressions with an implied 'any' type. */,
    // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */

    /* Additional Checks */
    "noUnusedLocals": true /* Report errors on unused locals. */,
    // "noUnusedParameters": true /* Report errors on unused parameters. */,
    // "noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
    "noFallthroughCasesInSwitch": true /* Report errors for fallthrough cases in switch statement. */,

    /* Module Resolution Options */
    "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
    "baseUrl": "." /* Base directory to resolve non-absolute module names. */,
    // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
    // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
    // "typeRoots": [],                       /* List of folders to include type definitions from. */
    // "types": [],                           /* Type declaration files to be included in compilation. */
    "allowSyntheticDefaultImports": true /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */,
    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
    // "preserveSymlinks": true,              /* Do not resolve the real path of symlinks. */
    // "allowUmdGlobalAccess": true,          /* Allow accessing UMD globals from modules. */

    /* Source Map Options */
    "sourceRoot": "." /* Specify the location where debugger should locate TypeScript files instead of source locations. */,
    "mapRoot": "." /* Specify the location where debugger should locate map files instead of generated locations. */,
    // "inlineSourceMap": true /* Emit a single file with source maps instead of having a separate file. */,
    // "inlineSources": true /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */,

    /* Experimental Options */
    "experimentalDecorators": true /* Enables experimental support for ES7 decorators. */,
    "emitDecoratorMetadata": true /* Enables experimental support for emitting type metadata for decorators. */,

    /* Advanced Options */
    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
  }
}
```

**Comment by anonrig** [comment]
@iyobo in case you didn't see

**Comment by iyobo** [comment]
@anonrig 
Sounds like something you are using in your controller is swallowing errors and not popping it up the stack. If whatever that is was properly pushing errors up the stack, your controller action would throw/return a boom error.

It is impossible for me to know what the case is without seeing some of your controller code.

**Comment by anonrig** [comment]
Hey @iyobo 
You should definitely check your Patreon :)

Here's my controller:

```typescript
import {
  Controller,
  Get,
  Post,
  Put,
  Delete,
  Flow,
  Ctx,
  Params
} from 'koa-ts-controllers'
import Koa from 'koa'
import Boom from '@hapi/boom'

import {
  isAuthenticated,
  shouldPaginate,
  hasRole,
  validateInput
} from '../middlewares'
import * as validations from './validations'
import { Account, Role } from '../models'

@Controller('/v1/accounts')
@Flow(isAuthenticated)
export class AccountController {}
```

**Comment by iyobo** [comment]
Good news @anonrig !
I just released v1.4 of koa-ts-controllers which now allows for loading controllers directly as classes. 

```typescript
import {FooController} from './controllers/FooController';
import {BarController} from './controllers/BarController';

await bootstrapControllers(app, {
        ...
        controllers: [FooController, BarController],
        ...
    });
```

And that's just for visibility reasons. In truth, you don't even really have to even add these to the bootstrap controllers array because simply importing them will have  the decorators in the controller class register itself appropriately i.e

```typescript
import {FooController} from './controllers/FooController';
import {BarController} from './controllers/BarController';

await bootstrapControllers(app, {
        ...
        controllers: [], // can be empty, can have a glob if you so choose and if it globs do not cause you issues.
        ...
    });
```
but uniformity

**Comment by anonrig** [comment]
Thanks @iyobo !
